test-data/unit/check-unreachable-code.test

```diff
@@ -404,9 +404,9 @@ x = 1
 [out]
 
 [case testCustomSysVersionInfo]
-# flags: --python-version 3.2
+# flags: --python-version 3.5
```

Comment:
I could add testcases to test-data/unit/cmdline.test like this:
```
[case testPython2VersionTooOld]
# cmd: mypy --python-version 2.6 -c pass
[out]
...
```
But a problem with that is that the output includes the full usage message, e.g.
```
  usage: mypy [-h] [-v] [-V] [--python-version x.y] [--platform PLATFORM] [-2] [--ignore-missing-imports] [--follow-imports {normal,silent,skip,error}] (diff)
              [--disallow-untyped-calls] [--disallow-untyped-defs] [--check-untyped-defs] [--disallow-subclassing-any] [--warn-incomplete-stub] [--warn-redundant-casts] (diff)
              [--no-warn-no-return] [--warn-return-any] [--warn-unused-ignores] [--show-error-context] [-i] [--quick-and-dirty] [--cache-dir DIR] [--strict-optional] (diff)
              [--strict-optional-whitelist [GLOB [GLOB ...]]] [--junit-xml JUNIT_XML] [--pdb] [--show-traceback] [--stats] [--inferstats] [--custom-typing MODULE] (diff)
              [--custom-typeshed-dir DIR] [--scripts-are-modules] [--config-file CONFIG_FILE] [--show-column-numbers] [--find-occurrences CLASS.MEMBER] [--strict] (diff)
              [--strict-boolean] [--cobertura-xml-report DIR] [--html-report DIR] [--linecount-report DIR] [--linecoverage-report DIR] [--memory-xml-report DIR] (diff)
              [--old-html-report DIR] [--txt-report DIR] [--xml-report DIR] [--xslt-html-report DIR] [--xslt-txt-report DIR] [-m MODULE] [-c PROGRAM_TEXT] [-p PACKAGE] (diff)
              [files [files ...]]               (diff)
  mypy: error: argument --python-version: Python 2.8 is not supported (must be 2.7) (diff)
```
which requires updating the tests frequently because the usage message changes each time we add a new flag.

Not sure I like tinkering with this more.